server/tso/global_allocator.go

```diff
@@ -104,77 +146,127 @@ func (gta *GlobalTSOAllocator) SetTSO(tso uint64) error {
 	return gta.timestampOracle.resetUserTimestamp(gta.leadership, tso, false)
 }
 
-// GenerateTSO is used to generate a given number of TSOs.
-// Make sure you have initialized the TSO allocator before calling.
+// GenerateTSO is used to generate the given number of TSOs.
+// Make sure you have initialized the TSO allocator before calling this method.
 func (gta *GlobalTSOAllocator) GenerateTSO(count uint32) (pdpb.Timestamp, error) {
```

Comment:
I suggest to write a detail summary for `GlobalTSOAllocator.GenerateTSO` to explain the difference process between enableGlobalTSOEstimation or not.